### PR TITLE
Add pre-commit CI in continuous integration

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
This permit to autofix pull request.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :sparkles: New feature |
| ✓   | :scroll: Docs          |

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -110,7 +110,7 @@ jobs:
         run: |
           . venv/bin/activate
           pip install -e .
-          pre-commit run --all-files
+          pre-commit pylint run --all-files
```
Comment (cdce8p) [inline .github/workflows/ci.yaml:113]:
```suggestion
          pre-commit run pylint --all-files
```

**Review by cdce8p — COMMENTED** [review]
I'm not completely sure this is actually what you want to do. With the current change it will only run the `pylint` pre-commit task. I guess you actually want to copy the existing one for formatting and then use `run pylint` for the new / copied one.

**Comment by Pierre-Sassoulas** [comment]
The other one are done in the ``This permit to autofix pull request.`` job, we need to disable pylint in it because pylint need to be installed as it's a system hook. So we only need to run the pylint one in the github action. The astroid MR is clearer : https://github.com/PyCQA/astroid/pull/982

**Comment by cdce8p** [comment]
> The other one are done in the `pre-commit.ci - pr — checks completed successfully .` job, we need to disable pylint in it because pylint need to be installed as it's a system hook. So we only need to run the pylint one in the github action. The astroid MR is clearer : [PyCQA/astroid#982](https://github.com/PyCQA/astroid/pull/982)

I see. Didn't noticed that you added that you installed that earlier.
Think I already asked that, but if you can install apps, aren't you also able to remove them (appveyor)?

**Comment by Pierre-Sassoulas** [comment]
No Appveyor do not have proper group authorization for PyCQA or I don't see where I can remove the job for pylint/astroid. I have no projects, no environment and nothing useful in my profil.

**Comment by cdce8p** [comment]
Someday 😅

**Pull request merged**